### PR TITLE
Fix writemime deprecation

### PR DESCRIPTION
**Pull request by malmaud**

### CODE REVIEW

File: src/LaTeXStrings.jl [hunk 1]
```diff
@@ -30,13 +30,14 @@ latexstring(args...) = latexstring(string(args...))
...
+@compat import Base.show
 
 write(io::IO, s::LaTeXString) = write(io, s.s)
-writemime(io::IO, ::MIME"application/x-latex", s::LaTeXString) = write(io, s)
-writemime(io::IO, ::MIME"text/latex", s::LaTeXString) = write(io, s)
+@compat show(io::IO, ::MIME"application/x-latex", s::LaTeXString) = write(io, s)
+@compat show(io::IO, ::MIME"text/latex", s::LaTeXString) = write(io, s)
 
-function show(io::IO, s::LaTeXString)
+@compat function show(io::IO, s::LaTeXString)
```
Comment (stevengj) [inline src/LaTeXStrings.jl]: Why is this necessary?  It's still `show` in older versions.

Comment (malmaud) [inline src/LaTeXStrings.jl]: Ya, guess I just got `@compat`-happy.

**Comment by stevengj** [comment]
Travis is failing for 0.3.  I think it should be fine to drop 0.3 support (update the REQUIRE and travis files).

File: REQUIRE [hunk 2]
```diff
@@ -1,2 +1,2 @@
-julia 0.3
+julia 0.4
```
Comment (stevengj) [inline REQUIRE]: You also need to update `.travis.yml`

Comment (malmaud) [inline REQUIRE]: It's updated
